plugins/panel/plugin.js

```diff
@@ -381,7 +381,7 @@
 					// Move forward.
 					case 'next':
 						var index = this._.focusIndex,
-							links = this.element.getElementsByTag( 'a' ),
+							links = this._.getItems(),
```

Comment:
These are no longer links. We should refer to this in a more generic way, like "focusables".

Reply:
This problem is repeated in couple other places.